apps/server/src/controllers/paymentsController/fundPersonalSavingsWallet.ts

```diff
@@ -0,0 +1,124 @@
+import { Response } from 'express'
+import { Op } from 'sequelize'
+import { HTTP_STATUS_CODE } from '../../constants'
+import { RequestExt } from '../../middlware/authorization/authentication'
+import Wallets, { WalletType, OwnerType } from '../../models/wallets'
+import { fundWalletSchema } from '../../utils/validators'
+import Transactions, {
+  transactionStatus,
+  transactionType,
+  action,
+} from '../../models/transactions'
+import { v4 } from 'uuid'
+
+export const fundPersonalSavingsWallet = async (
+  req: RequestExt,
+  res: Response
+) => {
+  try {
+    const { _userId: userId, ...rest } = req.body
+    const verityAmount = { amount: rest.amount }
```

Comment:
It's safe to write this within the safe parse:

`21    const reqData = fundWalletSchema.strict().safeParse({ amount: rest.amount })
`

also, since we are not taking a lot of data, this can be easily debugged, therefore, we can also do this (I think this should be fine in this context):

`21    const reqData = fundWalletSchema.safeParse(rest)
`